### DISCUSSION TITLE
Problems installing

### DISCUSSION THREAD

**Discussion by SilverFang13** [soulhotel/FF-ULTIMA#13] [Q&A] [answered]
I am probably doing something totally dumb but I can't seem to get the theme to work. I've attached pictures of my following the tutorial and then at the end I restart Firefox but nothing changes. Any help would be greatly appreciated
1
![Screenshot 2024-04-04 115424](https://github.com/soulhotel/FF-ULTIMA/assets/151080138/c184b400-4bfc-4727-b384-19cd251efc66)
2
![Screenshot 2024-04-04 115003](https://github.com/soulhotel/FF-ULTIMA/assets/151080138/74fc5c57-de23-453e-ac91-6f8c4420107f)
3
![Screenshot 2024-04-04 120026](https://github.com/soulhotel/FF-ULTIMA/assets/151080138/2ca96af4-f348-47ae-a65b-ead3e5377405)
4
![Screenshot 2024-04-04 120109](https://github.com/soulhotel/FF-ULTIMA/assets/151080138/4ba37300-b831-45cf-be07-1257542e61a6)
5
![Screenshot 2024-04-04 115737](https://github.com/soulhotel/FF-ULTIMA/assets/151080138/05a2f7cf-f3c5-4ac2-a0f8-b61bbf8a4328)

**Comment by Kaedriz** [comment, marked as answer]
You need to extract this folder.
 
Your `chrome` folder should look like this:
![Screenshot_20240404_193448](https://github.com/soulhotel/FF-ULTIMA/assets/44523955/479df88a-b8d2-44ef-b16b-6e4b7615d73a)
(Aside from optional.md and maybe .git)

Installation instructions have changed a bit today and need to be rewritten.

**Reply by soulhotel** [reply]
Yes, as he says above, everything INSIDE the ffultima1.6.7 folder goes inside of chrome. not the folder itself.
